### PR TITLE
Eloquent description

### PR DESCRIPTION
**Pull request by doronhi**
Add realsense2_description package.
Available models for D415, D435, D435i.
All models can be viewed using view_model.launch.py

### CODE REVIEW

File: realsense2_description/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.5)
+project(realsense2_description)
+
+find_package(ament_cmake REQUIRED)
+
+find_package(realsense2_camera REQUIRED)
+find_package(realsense2_node REQUIRED)
```
Comment (AustinDeric) [inline realsense2_description/CMakeLists.txt]: are these packages actually used in the realsense2_description package? (similar comment in package.xml)

File: realsense2_description/package.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <author email="sergey.dorodnicov@intel.com">Sergey Dorodnicov</author>
+  <author email="doron.hirshberg@intel.com">Doron Hirshberg</author>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+
+  <depend>realsense2_camera</depend>
+  <depend>realsense2_node</depend>
+  <depend>librealsense2</depend>
```
Comment (AustinDeric) [inline realsense2_description/package.xml]: I don't think this description package depends on `librealsense2`

File: realsense2_description/urdf/_d415.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      </collision>
+      <inertial>
+        <!-- The following are not reliable values, and should not be used for modeling -->
+        <mass value="0.564" />
+        <origin xyz="0 0 0" />
+        <inertia ixx="0.003881243" ixy="0.0" ixz="0.0" iyy="0.000498940" iyz="0.0" izz="0.003879257" />
+      </inertial>
+    </link>
+
+    <!-- Use the nominal extrinsics between camera frames if the calibrated extrinsics aren't being published. e.g. running the device in simulation  -->
```
Comment (AustinDeric) [inline realsense2_description/urdf/_d415.urdf.xacro:76]: line length

**Comment by AustinDeric** [comment]
@dorodnic looking good! i'd recommend making a foxy branch for this stuff because it works on that and in my opinion its one of the first minimum viable releases of ROS2 for industrial products. https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy/

**Comment by doronhi** [comment]
Thanks @AustinDeric , I was very pleased to have your remarks. Fixed accordingly.

**Pull request merged**